consensus/src/quorum_store/utils.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use consensus_types::common::Round;
+use std::{
+    cmp::Reverse,
+    collections::{BinaryHeap, HashSet},
+    hash::Hash,
+};
+
+#[allow(dead_code)]
+pub(crate) struct RoundExpirations<I: Ord> {
```

Comment:
It looks like this file isn't used in this PR? If so, please remove

Reply:
Yes, After the change this is no longer needed. I hoped nobody we notice though:)

Reply:
Removed.